### ISSUE TITLE
Can't edit feature

### ISSUE THREAD

**Issue by barseghyanartur** [pydanny/djangopackages#392]
URL: https://djangopackages.org/grids/element/983/1628/

```text
Edit Element which is related to:
Grid: Form builders
Package: django-forms-builder
Feature: Formsets Are formsets supported?
Text:
No
```

Output:
```
Error
If this seems like a bug, would you please do us a favor and create a ticket?
```

**Comment by EmadMokhtar** [comment]
I can't edit the documentation url as well

**Comment by Flimm** [comment]
I'm trying to edit the title of `django-notification`, and I also got an error: https://djangopackages.org/packages/django-notification/edit/

I also got a similar error when trying to add an example to it: https://djangopackages.org/packages/django-notification/example/add/

**Comment by ihoru** [comment]
Still don't work

**Comment by barseghyanartur** [comment]
It might be late already. It has been working badly for a long time. 

With http://libi.io/ offering a much broader spector (not tied to Django or Python; can be used for anything) this service (which I liked in past) becomes irrelevant.

**Comment by Flimm** [comment]
I personally continue to find djangopackages.org very useful, and it makes me sad to see it broken like this. Is there any way I can help at all?

**Comment by jayfk** [comment]
Not at this point, but I hope we are relatively close to finish the migration.

**Comment by jayfk** [comment]
Fixed in 86861a4 and backported to the "old" server.

**Comment by Flimm** [comment]
Yay! Everything seems to work again.
